packages/xdl/src/Simulator.js

```diff
@@ -219,6 +219,21 @@ async function _listSimulatorDevicesAsync() {
   return info;
 }
 
+async function _waitForDeviceToBoot() {
+  let bootedDevice;
+  const start = Date.now();
+  do {
+    await delayAsync(100);
+    bootedDevice = await _bootedSimulatorDeviceAsync();
+    if (Date.now() - start > 10000) {
+      Logger.global.error(
+        `iOS Simulator device failed to boot. Try opening Simulator first, then running your app.`
+      );
+      throw 'Timed out waiting for iOS Simulator device to boot.';
```

Comment:
```suggestion
      throw new Error('Timed out waiting for iOS Simulator device to boot.');
```
It's best to never throw strings, only errors.